### PR TITLE
this is the third file

### PR DESCRIPTION
**Pull request by maheswar92**
this is the third file for practice

### CODE REVIEW

File: test3.txt [hunk 1]
```diff
@@ -0,0 +1 @@
+this is the third test file
```
Comment (Mahicloud92) [inline test3.txt:1]: add second line here

Comment (maheswar92) [reply test3.txt:1]: added second line please review

Comment (Mahicloud92) [reply test3.txt:1]: evrything is good

**Review by Mahicloud92 — APPROVED** [review]
evrything seems good

**Comment by maheswar92** [comment]
approved

**Pull request merged**